packages/happo-target-firefox/src/waitForImagesToRender.js

```diff
@@ -1,7 +1,13 @@
 import findBackgroundImageUrls from './findBackgroundImageUrls';
 
-function waitForImageToLoad(url) {
+export function waitForImageToLoad(url) {
   return new Promise((resolve, reject) => {
+    // No need to wait for the image to be loaded.
+    if (url.indexOf('data:image') === 0) {
```

Comment:
I think we might want to be a little more specific with this search. In my experience this was working alright with JPEG data URIs and was actually needed in order for those to be fully rendered by the browser (although I wonder if we should switch to something like `createImageBitmap` instead). I think this might only be a problem when the data URI is an SVG.

I also wonder if this should be in the promise or if we should add a filter before this function call. Thoughts?

In the project you were trying to add this to, did you encounter this with an SVG data URI or something else?